postcss.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    plugins: {
+        'postcss-import': {},
+        tailwindcss: {},
+        autoprefixer: {},
+    }
+}
```

Comment:
We should add a .editorconfig to enforce new-line at the end of files.

Reply:
I see we already have `insert_final_newline` didnt work for me. [VSCode OSX, I will manually add these lines]